product_docs/docs/edbcloud/beta/getting_started/02_connect_cloud_account.mdx

```diff
@@ -29,12 +31,50 @@ Take note of the **Application (client) ID**, you need it to configure your EDB
 1. Assign the owner role to the application. See [Assign a role to the application](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#assign-a-role-to-the-application) for instructions. Enter the **Display name** of the Azure AD application in the **Select** field of the **Add role assignment** panel. See [Open the Add role assignment pane](https://docs.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal?tabs=current#step-2-open-the-add-role-assignment-pane) for instructions.
 
-
-
-
-
-
-
+### Create Azure Active Directory Application Using `create-spn`
+
+To simplify the process of creating an Azure AD Application, EDB provides the `create-spn` script for Azure API users. You can download it [here](https://github.com/EnterpriseDB/cloud-utilities/tree/main/azure).
+Before using the script ensure that the following utilities are installed on your machine:
+* [jq command-line JSON processor](https://stedolan.github.io/jq/)
+* [azure cli](https://docs.microsoft.com/en-us/cli/azure/) v2.26 or above
+
+#### create-spn Usage
+
+```
+Required permissions:
+  Microsoft.Authorization/roleAssignments/write
+Required tools:
+  jq
+Usage:
+  ./azure/create-spn.sh -d NAME -s SUBSCRIPTION_ID [options]
+
+Options:
+  -d, --display-name: The name of Azure AD App.
+  -s, --subscription: The Azure Subscription ID used by EDB Cloud.
+  -h, --help:         Show this help.
+```
+
+#### Example 
+
+To create an Azure AD Application with the subscription ID  `c808xxxx-xxxx-xxxx-xxxx-xxxxxxxxb712` and display name `hello-s`:
+
+```
+./create-spn.sh -d "hello-s" -s c808xxxx-xxxx-xxxx-xxxx-xxxxxxxxb712
+Change to use Azure Subscription c808xxxx-xxxx-xxxx-xxxx-xxxxxxxxb712...
+EnvironmentName    HomeTenantId                          IsDefault    Name         State    TenantId
+-----------------  ------------------------------------  -----------  -----------  -------  ------------------------------------
+AzureCloud         fb0cxxxx-xxxx-xxxx-xxxx-xxxxxxxx3d0f  True         development  Enabled  fb0cxxxx-xxxx-xxxx-xxxx-xxxxxxxx3d0f
+Creating Azure AD Service Principal and configuring its access to Azure resources in subscription c808xxxx-xxxx-xxxx-xxxx-xxxxxxxxb712...
+WARNING: Creating 'Owner' role assignment under scope '/subscriptions/c808xxxx-xxxx-xxxx-xxxx-xxxxxxxxb712'
+
+WARNING: The output includes credentials that you must protect. Be sure that you do not include these credentials in your code or check the credentials into your source control. For more information, see https://aka.ms/azadsp-cli
+WARNING: 'name' property in the output is deprecated and will be removed in the future. Use 'appId' instead.
```

Comment:
What exactly is this warning, @fisher046?

Reply:
@ba-edb this warning is from Azure CLI. I think this is because the output contains secret value so the CLI gives that warning. We can add an argument `--only-show-errors` to Azure CLI to mute this, but I think it will hide some information from Azure..How do you think about it?